src/app/examples/nativescript-only/list-section-example/list-section-example.component.tns.ts

```diff
@@ -1,13 +1,20 @@
 import { Component } from '@angular/core';
 
-import { BaseListComponent } from '../../base-list.component';
+import { BaseListComponent } from '~/app/examples/list/base-list.component';
 
 @Component({
+  selector: 'kirby-list-section-example',
   templateUrl: './list-section-example.component.tns.html',
   styleUrls: ['./list-section-example.component.tns.scss'],
 })
 export class ListSectionExampleComponent extends BaseListComponent {
   getSectionName(item: any): string {
-    return item.detail > 0 ? 'Positive' : 'Negative';
+    if (item.detail > 0) {
+      return 'Positive';
+    } else if (item.detail < 0) {
+      return 'Negative';
+    } else {
+      return 'Nul';
```

Comment:
??

Reply:
Leftovers from testing variations of list... reverting back to previous example-implementation